### PR TITLE
Added changelist functionality to log_default() function

### PR DESCRIPTION
**Pull request by kris1250**

### CODE REVIEW

**Comment by kris1250** [comment]
bump

File: svn/common.py [hunk 1]
```diff
@@ -214,15 +214,21 @@ def log_default(self, timestamp_from_dt=None, timestamp_to_dt=None,
...
+
         result = self.run_command(
                     'log', 
                     args + ['--xml', full_url_or_path], 
                     combine=True)
 
         root = xml.etree.ElementTree.fromstring(result)
+        named_fields = ['date', 'msg', 'revision', 'author']
+        if changelist is True:
+            named_fields += ['changelist']
```
Comment (dsoprea) [inline svn/common.py]: Put the "changelist" item in there no matter what. Just set it to None if not requested. We want an immutable type to always have the same shape.

**Comment by mheppner** [comment]
bump

**Comment by kris1250** [comment]
Finished, sorry it took so long to make such a small change.

**Comment by dsoprea** [comment]
Thank you and come again.

**Pull request merged**